docs/conf.py

```diff
@@ -94,7 +94,7 @@ def check_if_v046():
 
 # Multiversion settings
 
-smv_tag_whitelist = r'^v0\.4\.6|^v0\.5\.\d|^v0\.6\.\d+$'
+smv_tag_whitelist = r'^v0\.4\.6|^v0\.5\.\d|^v0\.6\.2|^v0\.7\.\d+$'
```

Comment:
could we change this to a regex accepting any versions?

Reply:
Yes, but I consciously don't want to accept all versions, only the highest patch version of each minor (see PR description). Maybe that's possible to do with a regex (?)
However, I felt like it's easier/safer/more intuitive to write a little bit of code to generate this regex (added in last couple commits). Wdyt?

Reply:
The PR includes the extra code now.